### PR TITLE
Module split

### PR DESCRIPTION
**Pull request by Pessimistress**
Modules:
- `probe.gl`
- `@probe.gl/bench`
- `@probe.gl/test-utils`

The following scripts are tested:
- `bootstrap`
- `build`
- `clean`
- `test`
- `lint`

Will test `publish` once merged.

### CODE REVIEW

File: modules/bench/src/format-utils.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export function formatSI(number, precision = 3) {
```
Comment (ibgreen) [inline modules/bench/src/format-utils.js]: This diffs as a new file. Is this file being copied or was the intention to move it?

I assume new code is not being added in this restructuring PR, better keep that for a separate PR>

File: modules/test-utils/src/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-export {makeSpy} from './make-spy';
+export {makeSpy} from './utils/make-spy';
```
Comment (ibgreen) [inline modules/test-utils/src/index.js]: `make-spy` is not a util, it is a major exported function/symbol from test-utils.

File: modules/bench/src/format-utils.js (hunk 1 above)
Comment (Pessimistress) [reply modules/bench/src/format-utils.js]: This is moved from `formatters.js`. `Bench` is the only class depending on this so it doesn't make sense to export it from core.

**Comment by Pessimistress** [comment]
The latest ocular-dev-tools work on OSX but fails on Travis. I will land this and follow up with a fix.
